docs/user/troubleshooting.md

```diff
@@ -27,3 +27,36 @@ sudo flightctl-must-gather
 ```
 
 The output is a tarball named `must-gather-$timestamp.tgz`, whereby `$timestamp` is the current date and time. Include this tarball in the bug report.
+
```

Comment:
We can remove the must-gather section above.

Reply:
It still exists.

Reply:
It should be obsolete with `sos report` now, so we should remove from docs and RPM package.

Reply:
OK